packages/desktop-client/src/components/reports/ReportSidebar.tsx

```diff
@@ -297,12 +301,12 @@ export function ReportSidebar({
             </Text>
             <Select
               value={customReportItems.sortBy}
-              onChange={e => onChangeSortBy(e)}
+              onChange={(e?: sortByOpType) => onChangeSortBy(e)}
               options={ReportOptions.sortBy.map(option => [
-                option.description,
+                option.format,
```

Comment:
Good catch. The migration (1736640000000__custom_report_sorting.sql) will need changing/another adding to fix "Descending" to "desc".